app/referee/tasks.py

```diff
@@ -1,74 +1,152 @@
 import json
 import logging
 import math
+from collections import namedtuple
 
 import dask
 import dask.distributed as dd
-
 import ever_given.wrapper
 
 from core import models
 
-
 logger = logging.getLogger(__name__)
 
 
+AnswerPredictionPair = namedtuple("AnswerPredictionPair", ["answer", "prediction"])
+
+SIMPLE = "SIMPLE"
+MULTI = "MULTI_JSON"
+
+
+def score_evaluation(container, evaluation, evaluation_score_types, output_handling):
+    if len(evaluation_score_types) == 1:
+        evaluation_score_handling = SIMPLE
+        evaluation_score_type = list(evaluation_score_types.values())[0]
+    else:
+        evaluation_score_handling = MULTI
+
+    predictions = {
+        prediction.value_type.key: prediction.value
+        for prediction in evaluation.prediction_set.all()
+    }
+    input_element = evaluation.input_element
+
+    answer_keys = {
+        answer_key.value_type.key: answer_key.value
+        for answer_key in input_element.answerkey_set.all()
+    }
+
+    # if logger.isDebugEnabled():
+    # smiles_type = challenge.valuetype_set.get(key="SMILES")
+    # input_value = input_element.inputvalue_set.first()
+    # get(input_type=smiles_type)
+    #    logger.debug(
+    #        "Prediction: %s %s answer: %s",
+    #        prediction.value,
+    #        input_value.value,
+    #        answer_key.value,
+    #    )
+
+    command = None
+    score_raw_args = {
+        key: AnswerPredictionPair(answer_value, predictions[key])
+        for key, answer_value in answer_keys.items()
+    }
+
+    if output_handling == SIMPLE:
+        assert len(score_raw_args) == 1, "More than one output type for SIMPLE!"
+        for ap_pair in score_raw_args.values():
+            command = "{} {} ".format(ap_pair.answer, ap_pair.prediction)
```

Comment:
```suggestion
            command = f"{ap_pair.answer} {ap_pair.prediction} "
```
Any reason to not use an `f-string` here?